geopyspark/geopycontext.py

```diff
@@ -56,6 +56,8 @@ def __init__(self, pysc=None, **kwargs):
         self.sc = self.pysc._jsc.sc()
         self._jvm = self.pysc._gateway.jvm
 
+        self.pysc._gateway.start_callback_server()
```

Comment:
Do you think it would be better to start the callback server only when someone is trying to a tms server?